src/Worker.php

```diff
@@ -80,9 +80,11 @@ private function tick($queue)
         }
 
         try {
+            $this->event->acknowledge($message);
```

Comment:
As in outside of the `try` / `catch` block? What happens if a handler throws an exception?